Inspicio/Inspicio.csproj

```diff
@@ -20,12 +20,26 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Content Include="wwwroot\css\home.css" />
     <Content Include="wwwroot\css\JavaScript.js" />
     <Content Include="wwwroot\css\thumbs.js" />
+    <Content Include="wwwroot\images\add.svg" />
+    <Content Include="wwwroot\images\create.svg" />
+    <Content Include="wwwroot\images\eye.jpg" />
+    <Content Include="wwwroot\images\Inspicio bar.png" />
```

Comment:
Spaces in names of files served by the web server can be problematic, typically people use `-` instead. e.g. in this url https://stackoverflow.com/questions/497908/is-a-url-allowed-to-contain-a-space.

You can also run into problems with some web servers being case sensitive, so it's also common to use lowercase letters for everything.